### PR TITLE
Vcon 280 conversation window

### PR DESCRIPTION
**Pull request by DaceyTom2**
### Description
ConversationWindow test portion of VCON-280 using shallow mount, i18n plugin and mocked store
Please let me know if there are other opportunities to create more robust tests

### Additional Notes
Added lint to test files
Removed console logs from tests
Example of button click left for use in smaller components
Store mocked
i18n plugin added

### CODE REVIEW

**Review by shewood — APPROVED** [review]
This looks good.

File: __tests__/components/organisms/ConversationWindow.spec.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+  it("Check for text input component", () => {
+    expect(wrapper.findComponent(TextInput).exists()).toBe(true);
+  });
+
+  it("Test sendChatMessage", async () => {
+    // const textInput = wrapper.find('#textInputInput')
+    // await textInput.setValue('test value')
+    // const sendTextInputButton = wrapper.find("#sendTextInputButton");
+    // sendTextInputButton.trigger('click');
```
Comment (nibivi77) [inline __tests__/components/organisms/ConversationWindow.spec.js:49]: Should we keep these comments?

File: __tests__/components/organisms/ConversationWindow.spec.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+import { createStore, useStore } from "vuex";
+import { i18n } from "../../../i18n";
+import chatMessages from "../../../src/store/modules/chatMessages";
+jest.mock("../../../src/store/modules/chatMessages");
+import emails from "../../../src/store/modules/emails";
+jest.mock("../../../src/store/modules/emails");
+import inbox from "../../../src/store/modules/inbox";
+jest.mock("../../../src/store/modules/inbox");
+
+describe("ConversationWindow component", () => {
```
Comment (nibivi77) [inline __tests__/components/organisms/ConversationWindow.spec.js:14]: Could we check to make sure that the `ConversationMessage` component is rendered as well? Just like we did with line 37 and 41.

**Review by nibivi77 — COMMENTED** [review]
This is some really cool stuff. Mocking the store. Great work ! I left two comments.

File: __tests__/components/organisms/ConversationWindow.spec.js (hunk 1 above)
Comment (DaceyTom2) [reply __tests__/components/organisms/ConversationWindow.spec.js]: I wanted to keep as an example but I've already done in another PR I've removed them

File: __tests__/components/organisms/ConversationWindow.spec.js (hunk 2 above)
Comment (DaceyTom2) [reply __tests__/components/organisms/ConversationWindow.spec.js:15]: Added check for ConversationMessages. Quite beneficial as it checks if getter returns value that v-for can render ConversationMessages

**Pull request merged**